com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/aps/ZigBeeApsFrame.java

```diff
@@ -78,6 +78,12 @@
      */
     private boolean securityEnable;
 
+    /**
+     * The AckRequest parameter specifies whether the current transmission requires an acknowledgement frame to be sent
+     * be the recipient on receipt of the frame.
+     */
+    private boolean ackRequest;
```

Comment:
One note: It seems that Telegesis API doesn't support disabling ACK request in the APS frame. Other dongles looks like they do but I only had a chance to test on CC2531